sklearn/ensemble/tests/test_bagging.py

```diff
@@ -748,3 +749,99 @@ def test_set_oob_score_label_encoding():
     x3 = BaggingClassifier(oob_score=True,
                            random_state=random_state).fit(X, Y3).oob_score_
     assert_equal([x1, x2], [x3, x3])
+
+
+def test_bagging_regressor_with_missing_inputs_and_1d_output():
+    # Check that BaggingRegressor can accept X with missing/infinite data
+    X = [
+        [1, 3, 5],
+        [2, None, 6],
+        [2, np.nan, 6],
+        [2, np.inf, 6],
+        [2, np.NINF, 6],
+    ]
+    Y = [2, 3, 3, 3, 3]
+    regressor = DecisionTreeRegressor()
+    pipeline = make_pipeline(
+        Imputer(),
+        Imputer(missing_values=np.inf),
+        Imputer(missing_values=np.NINF),
+        regressor
+    )
+    pipeline.fit(X, Y).predict(X)
+    bagging_regressor = BaggingRegressor(pipeline)
+    bagging_regressor.fit(X, Y).predict(X)
+
+    # Verify that exceptions can be raised by wrapper regressor
+    regressor = DecisionTreeRegressor()
+    pipeline = make_pipeline(regressor)
+    assert_raises(ValueError, pipeline.fit, X, Y)
+    bagging_regressor = BaggingRegressor(pipeline)
+    assert_raises(ValueError, bagging_regressor.fit, X, Y)
```

Comment:
Might be good to check the message here, but okay.

Reply:
Since this is just a test for BaggingRegressor, I don't really want to couple the behavior of the underlying regressor to this test case. The fact that it raises an error seems sufficient to me.